src/platform/mod.rs

```diff
@@ -2,7 +2,7 @@
 #[cfg(any(target_os = "linux", target_os = "freebsd"))]
 pub mod linux;
 /// MacOs specific stuff
-#[cfg(target_os = "macos")]
```

Comment:
Two more things:

1. Could you go ahead and rename the `macos` module to `apple` as well?
This might get confusing otherwise.

2. Can we properly test this in the CI? There probably are no OS target provided by Github, aren't there?
3.

Reply:
Hey! I'll get these changes done tonight or tomorrow morning, you can test this by adding a --target flag to cargo with aarch64-apple-ios for ios, x86_64-apple-darwin for intel macs, etc.